src/style.css

```
@@ -1 +1,44 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars :last-child {
+  margin-right: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(/src/images/star.svg);
+}
+
+.stars--0 :nth-child(-n + 0) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url(/src/images/star-active.svg);
+}
```

```suggestion
.stars--0 :nth-child(-n + 0),
.stars--1 :nth-child(-n + 1) {
  background-image: url(/src/images/star-active.svg);
}
```
you can combine it all using the comma

---

is it possible to insert a variable into nth-child?